docs/modules/ROOT/pages/advanced-how-tos/installing/index.adoc

```diff
@@ -1 +1,83 @@
-= Installing {ProductName}
\ No newline at end of file
+= Installing {ProductName}
+
+{ProductName} offers a comprehensive solution that fortifies your software supply chain against emerging threats. By providing detailed insights into the composition and construction of your software, {ProductName} enables proactive identification and mitigation of vulnerabilities.
+
+Before tapping into the vast array of benefits offered by {ProductName}, the initial step involves its installation. The installation of {ProductName} is structured around several key procedures:
+
+== Trying Out {ProductName}
+
+This section demonstrates the process for deploying {ProductName} locally, onboarding users, and building and releasing an application. Users have two options for onboarding applications to {ProductName}:
+
+. *Using the {ProductName} UI*: This option provides a more streamlined user experience but requires using Quay.io for the image registry and additional initial setup steps.
+
+. *Using Kubernetes manifests*: This option supports any image registry and involves fewer initial setup steps.
+
+=== Prerequisites
+
+* You must have Platform Engineers (PEs) role.
+
+* Machine Minimum Requirements:
+
+** CPU: 4 cores
+** RAM: 8 GB
+
+* Software Dependencies:
+
+** Kind and kubectl, along with podman or docker
+
+** git
+
+** openssl
+
+=== Installation Steps
+
+. *Bootstrapping the Cluster*:
++
+[source,sh]
+----
+kind create cluster --name {ProductName} --config kind-config.yaml
```

Comment:
Should we specifically document installation on Kind here?
We have documentation for installing on Kind in the konflux-ci repo as there we're inviting people to try it out.
If I understand correctly, the intention here is to provide something more generic (correct me if I'm wrong). if so, I'd expect something less specific to the nature of the underlying cluster.